server/controllers/users.js

```diff
@@ -34,14 +34,17 @@ export const exists = async (req, res) => {
 export const signin = (req, res, next) => {
   const { user, redirect, websiteUrl } = req.body;
   let loginLink;
+  let clientIP;
   return models.User.findOne({ where: { email: user.email.toLowerCase() } })
     .then(u => u || models.User.createUserWithCollective(user))
     .then(u => {
       loginLink = u.generateLoginLink(redirect || '/', websiteUrl);
+      clientIP = req.headers['x-forwarded-for'] || req.connection.remoteAddress;
       if (config.env === 'development') {
         logger.info(`Login Link: ${loginLink}`);
+        logger.info(`Client IP: ${clientIP}`);
```

Comment:
That should not be necessary. We log the `loginLink` not for troubleshooting but to use it to login! This is why it's relevant.